src/nvim/strings.c

```diff
@@ -455,23 +455,19 @@ char_u *vim_strchr(const char_u *const string, const int c)
   }
 }
 
-/*
- * Search for last occurrence of "c" in "string".
- * Return NULL if not found.
- * Does not handle multi-byte char for "c"!
- */
+/// strrchr() version which does not handle multi-byte char for "c"!
+///
+/// @param[in]  string  String to search in.
+/// @param[in]  c  Character to search for.
+///
+/// @return Pointer to the last occurrence in string or NULL
+///         if it was not found. NUL character is not
+///         handled, use `strlen()` instead.
 char_u *vim_strrchr(const char_u *string, int c)
```

Comment:
We could replace the few uses of `vim_strrchr()` with `strrchr()` directly and remove this function, because we always use `c<0x80`.

Reply:
That would be the preferable choice IMO - as long as it is not exposed as an interface.